include/targets/cuda/mdw_dslash5_tensor_core.cuh

```diff
@@ -367,7 +366,8 @@ namespace quda
     a = __hmul2(sm_b[(threadIdx.y * 4 + 3) * N_sm_d2 + 3 * threadIdx.x + 1], max_i_div_max2_);
     b = __hmul2(sm_b[(threadIdx.y * 4 + 3) * N_sm_d2 + 3 * threadIdx.x + 2], max_i_div_max2_);
     out[sid + 5 * output.volumeCB] = __2half22integer4_rn<storage_vec>(a, b);
-
+#else
+    static_assert(QUDA_ORDER_FP == 8 || QUDA_ORDER_FP == 4, "Invalid QUDA_ORDER_FP");
```

Comment:
I guess we do not need this check here, since it has already been checked generally in the other file?

Reply:
Agreed.  Changed here: 5357496edb2acea31f5d50614e6b3036d04299e7